pkgs/by-name/al/aliae/package.nix

```diff
@@ -0,0 +1,57 @@
+{
+  lib,
+  stdenv,
+  buildGoModule,
+  fetchFromGitHub,
+  installShellFiles,
+}:
+
+buildGoModule rec {
+  pname = "aliae";
+  version = "0.22.1";
+
+  src = fetchFromGitHub {
+    owner = "jandedobbeleer";
+    repo = "aliae";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-slixB7mzEdX3ecgbM6tO9IzVH+1w6DwssD1X3MrwAHw=";
+  };
+
+  vendorHash = "sha256-U0Mt2U8WxDFDadIxASz609tUtiF4tETobAmYrk29Lh0=";
+
+  sourceRoot = "${src.name}/src";
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.Version=${version}"
+  ];
+
+  tags = [
+    "netgo"
+    "osusergo"
+    "static_build"
```

Comment:
I’m not sure if we want this. Does it break without?

Reply:
I copied it from https://github.com/JanDeDobbeleer/aliae/blob/main/src/.goreleaser.yml